mithril-stm/src/participant/mod.rs

```diff
@@ -0,0 +1,7 @@
+//! Mithril-stm participant including StmInitializer and StmSigner
+
+mod initializer;
+mod signer;
+
+pub use crate::participant::initializer::{StmInitializer, StmVerificationKeyPoP};
+pub use crate::participant::signer::{StmSigner, StmVerificationKey};
```

Comment:
You don't need to specify the full paths as the `mod xxxx` already put them in scope :)
```suggestion
pub use initializer::{StmInitializer, StmVerificationKeyPoP};
pub use signer::{StmSigner, StmVerificationKey};
```